src/industry_gui.cpp

```diff
@@ -1317,25 +1317,53 @@ class IndustryDirectoryWindow : public Window {
 		static CargoSuffix cargo_suffix[lengthof(i->produced_cargo)];
 		GetAllCargoSuffixes(CARGOSUFFIX_OUT, CST_DIR, i, i->type, indsp, i->produced_cargo, cargo_suffix);
 
-		/* Industry productions */
+		/* Get sorted industry productions */
+		struct Cargo {
+			CargoID cargo;
+			uint16 production;
+			char *suffix;
+			uint transported;
+
+			/* Comparator to sort by descending production, then descending transported */
+			bool operator< (const Cargo &other) const
```

Comment:
this function appears to actually be implementing `>`... Confusing at best, incorrect at worst? Perhaps implement `>` (or `>=`) then define `<` in terms of that?